packages/core/bin/stryker

```diff
@@ -5,5 +5,5 @@
 process.title = 'stryker';
 
 // Run the Stryker CLI
-const StrykerCli = require('../src/StrykerCli').default;
+const { StrykerCli } = require('../src/StrykerCli');
```

Comment:
This was the issue. We used to export `StrykerCli` as default. Stryker compiled as `default` export. Now a named export.